encrypted-media-respec.html

```diff
@@ -2044,15 +2044,19 @@
                                 <ol>
                                   <li>
                                     <p>
-                                      If the <var>record of key usage</var> does not contain an entry for <var>block key ID</var>, add a new entry for <var>block key ID</var> with <var>first decrypt time</var> set to the current time.
+                                      If the <var>record of key usage</var> does not contain an entry for <var>block key ID</var>, add a new entry for <var>block key ID</var> with <var>first decrypt time</var> set to the current time, accurate to within <a def-id="key-usage-accuracy"></a>.
                                     </p>
                                   </li>
                                   <li>
                                     <p>
-                                      Set the <var>latest decrypt time</var> of the entry for <var>block key ID</var> in the <var>record of key use</var> to the current time.
+                                      Set the <var>latest decrypt time</var> of the entry for <var>block key ID</var> in the <var>record of key use</var> to the current time, accurate to within <a def-id="key-usage-accuracy"></a>.
```

Comment:
Perhaps this should be something like `Ensure the <var>latest decrypt time</var>... is accurate to within <a def-id="key-usage-accuracy"></a> of the current time.`